pelican/settings.py

```diff
@@ -163,7 +163,7 @@ def load_source(name, path):
     'WRITE_SELECTED': [],
     'FORMATTED_FIELDS': ['summary'],
     'PORT': 8000,
-    'BIND': '127.0.0.1',
+    'BIND': '127.0.0.1'
```

Comment:
Unless this change is somehow related to the other changes in this pull request, I suggest that this change be reverted.